packages/ember-metal/lib/is_proxy.js

```diff
@@ -1,7 +1,10 @@
-import { symbol } from 'ember-utils';
-
-export const IS_PROXY = symbol('IS_PROXY');
+import { peekMeta } from './meta';
 
 export function isProxy(value) {
-  return typeof value === 'object' && value && value[IS_PROXY];
+  if (typeof value === 'object' && value) {
+    let meta = peekMeta(value);
+    return typeof meta === 'object' && meta !== null && meta.isProxy();
```

Comment:
Right, I was asking if we could just check for either === null or undefined (whatever it actually returns when there is no meta) instead of he generic check we usually do.